### ISSUE TITLE
Move preflight check for webpush into validation.py

### ISSUE THREAD

**Issue by bbangert** [mozilla-services/autopush#563]
Once #492 is complete, then the preflight check from router/webpush.py should be moved into the WebPushRequest schema in validation.py.

**Comment by bbangert** [comment]
Blocked on #492.

**Comment by jrconlin** [comment]
crap, restoring because of blockage.

**Comment by pjenvey** [comment]
#765 was a duplicate of this -- now fixed as of PR #772
